etc/kayobe/kolla.yml

```diff
@@ -126,9 +126,8 @@ kolla_build_blocks:
     RUN grafana-cli plugins install vonage-status-panel
     # NOTE(brtknr): Install upstream patch currently under review:
     # https://review.opendev.org/#/c/670316/
-    RUN git clone https://github.com/stackhpc/monasca-grafana-datasource --depth=1 -b stackhpc/1.3.0 \
-        && grafana-cli --pluginsDir monasca-grafana-datasource/ plugins install monasca-datasource \
-        && rm -rf monasca-grafana-datasource/
+    RUN grafana-cli plugins remove monasca-datasource \
+        && git clone https://github.com/stackhpc/monasca-grafana-datasource --depth=1 -b stackhpc/1.3.0 /var/lib/grafana/plugins/monasca-datasource/
```

Comment:
nit: Rather than git clone a more canonical approach is to download the tar file, for example:

```
RUN curl -sSL -o /tmp/elasticsearch_exporter.tar.gz https://github.com/justwatchcom/elasticsearch_exporter/releases/download/v${elasticsearch_exporter_version}/elasticsearch_exporter-${elasticsearch_exporter_version}.linux-{{debian_arch}}.tar.gz \
    && tar xvf /tmp/elasticsearch_exporter.tar.gz -C /opt/ \
    && rm -f /tmp/elasticsearch_exporter.tar.gz \
    && ln -s /opt/elasticsearch_exporter* /opt/elasticsearch_exporter
```

Reply:
Hmm, that feels like a lot of additional steps to achieve the same goal. Notice that I have used a depth of 1 as a parameter so its not cloning the entire history.

Reply:
Indeed, I think the main reason is that you don't depend on git being present in the image. I forget why it is there actually, it looks like it could be removed from monasca_grafana Dockerfile.